app/controllers/application_controller.rb

```diff
@@ -14,6 +15,16 @@ def root
 
   private
 
+  def authenticate
+    sign_in(model_by_param(User, :email))
+  rescue
+    'Model cannot be authenticate'
+  end
+
+  def model_by_param(model, param)
```

Comment:
For universality
Function from object params
It is better to call the function model_by_params, It will be clearer